src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './App.css';
+import ProfilePicture from './conponents/Picture';
+import Header from './conponents/Header';
+import InfoCard from './conponents/Info';
+import SocialCard from './conponents/Social';
+
+const App = () => (
+  <>
+    <ProfilePicture />
+    <Header />
```

Comment:
Please try to add the **Profile Component** in the header. Because when you try to inspect your code then you will see code is actually going against the `html-semantics`.  Always keep in mind Follow a good hierarchy 👍🏻

Reply:
Ow sure! an `h1` tag should come top of the page as much as possible. I just implemented this recommendation.